src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -152,5 +165,25 @@ public virtual IValue ToAST(object value)
                 return null;
             }
         }
+
+        protected internal object ThrowLiteralConversionError(IValue input)
+        {
+            throw new ArgumentException($"Unable to convert '{input}' to '{Name}'");
```

Comment:
```suggestion
            throw new ArgumentException($"Unable to convert '{input}' from AST representation to '{Name}'");
```

Reply:
👍 

Once I can get existing tests to pass again, then I'll commit the revised exception messages.